packages/@okta/i18n/src/properties/login.properties

```diff
@@ -1276,6 +1276,7 @@ oie.user.profile.division = Division
 oie.user.profile.department = Department
 oie.user.profile.managerId = Manager ID
 oie.user.profile.manager = Manager
+oie.selfservice.unlock_user.success.message.with.application.name = Account successfully unlocked!<br>Continue to access {0}.
```

Comment:
@IanHakes-okta could you please take a look to this? here is the [figma](https://www.figma.com/file/fZGJS3FV0RpfKImY8CKwij/Magic-Link?node-id=2009%3A57303)